annotations/templates/annotations/header.html

```diff
@@ -111,28 +109,37 @@
                     </li>
                 </ul>
                 <ul class="nav navbar-nav navbar-right">
-                    <li class="{% if user.is_authenticated %}dropdown{% endif %}">
-                        {% if user.is_authenticated %}
-                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
-                                <span class="glyphicon glyphicon-dashboard"></span> <span class="caret"></span>
-                            </a>
-                            <ul class="dropdown-menu">
-                                <li><a href="{% url "dashboard" %}">Dashboard</a></li>
-                                <li><a href="{% url "settings" %}">Settings</a></li>
-                                <li><a href="{% url "logout" %}">Log Out</a></li>
-                                {% if user.is_staff %}
-                                    <li><a href="{{ base_url }}/admin/">Admin Site</a></li>
-                                {% endif %}
-                            </ul>
-                        {% else %}
-                            <a class="btn btn-sm" href="{% url 'account_login' %}" data-toggle="tooltip" title="to get started!" data-container="body" data-placement="bottom">
-                                <i class="fa fa-sign-in" aria-hidden="true"></i> {% trans 'Log in' %}
-                            </a>
-                        {% endif %}
-                    </li>
-                </ul>
-            </div><!--/.navbar-collapse -->
-        </div><!--/.container-fluid -->
```

Comment:
what happened to this div (container-fluid)? Is it not needed? Because the opening div tag is still there but it's never closed